performant-software/Annotation-Studio
# Editable docs

## What does this PR do?
1. Makes it so that users can not annotate documents that are editable.

## How to test?
1. Login and go to http://cove-staging.herokuapp.com/documents and click on a document that is editable
2. You should not be able to annotate a document and should see no errors in the javascript console.
